providers/fetcher/githubProcessor.js

```diff
@@ -248,14 +254,23 @@ class GitHubProcessor {
     return document;
   }
 
-  commit_comment(request) {
+  pull_request_commit_comment(request) {
+    return this.commit_comment(request, true);
+  }
+
+  commit_comment(request, isPullRequest = false) {
     // TODO links to consider
     // * reactions -- get this by using the following Accept header: application/vnd.github.squirrel-girl-preview
     const document = request.document;
     const context = request.context;
     request.addSelfLink();
-    request.linkResource('commit', context.qualifier);
-    request.linkSiblings(`${context.qualifier}:commit_comments`);
+    if (isPullRequest) {
```

Comment:
I prefer the pattern you did above where you used a variable containing the prefix and just conditioned setting that variable.